### ISSUE TITLE
negative boxes score high

### ISSUE THREAD

**Issue by DANNALI35** [TreB1eN/InsightFace_Pytorch#78]
Hi there,  i am re-implementing dual shot using keras. During testing,  my positive boxes scores are high, but the negative boxes also score quite high, **NMS in this case cannot filter boxes so well**, can someone enlighten me why this happens?

During training, i calculate the loss using all positive anchors with iou>0.4, negative anchors with iou<0.3. negative anchors are selected using hard negative mining, they are 3 times the number of positive anchors.

During testing, i use all initialized anchors to do a forward pass and do NMS on all anchors
